From 9abf1d89684b07473b05a0efd61b93a8f09ddb39 Mon Sep 17 00:00:00 2001
From: Ankush Pathak <ankush.pathak@chainguard.dev>
Date: Mon, 22 Sep 2025 07:43:49 +0000
Subject: [PATCH] fix: Don't use jna SNAPSHOT jar instead use release

net.java.dev.jna:jna dependency pulls in 5.13.0 SNAPSHOT jar
Explicitly request the release jar
---
 zipkin-server/pom.xml | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/zipkin-server/pom.xml b/zipkin-server/pom.xml
index c9359ef10..e57596c30 100644
--- a/zipkin-server/pom.xml
+++ b/zipkin-server/pom.xml
@@ -575,6 +575,13 @@
         <groupId>org.springframework.boot</groupId>
         <artifactId>spring-boot-maven-plugin</artifactId>
         <version>${spring-boot.version}</version>
+        <dependencies>
+          <dependency>
+            <groupId>net.java.dev.jna</groupId>
+            <artifactId>jna</artifactId>
+            <version>5.13.0</version>
+          </dependency>
+        </dependencies>
         <configuration>
           <mainClass>zipkin.server.ZipkinServer</mainClass>
           <executable>true</executable>
-- 
2.51.0

